frame/staking/src/pallet/impls.rs

```diff
@@ -59,6 +59,38 @@ use super::{pallet::*, STAKING_ID};
 const NPOS_MAX_ITERATIONS_COEFFICIENT: u32 = 2;
 
 impl<T: Config> Pallet<T> {
+	/// Returns the current nominations quota for nominators.
+	///
+	/// Used by runtime API.
+	/// Note: for now, this api runtime will always return value of `T::MaxNominations` and thus it
+	/// is redundant. However, with the upcoming changes in https://github.com/paritytech/substrate/pull/12970
+	/// the nominations quota will change depending on the nominators balance. We're introducing
+	/// this runtime API now to prepare the community to use it before rolling out PR#12970.
+	pub fn api_nominations_quota(_balance: BalanceOf<T>) -> u32 {
+		T::MaxNominations::get()
+	}
+
+	/// Returns the current inflation rate.
+	///
+	/// The staked amount as a PerThing is calculated and provided to `compute_inflation` alongside
+	/// the provided `ideal_staked` and `falloff` values.
+	pub fn api_inflation_rate(ideal_staked: Perquintill, falloff: Perquintill) -> Perquintill {
```

Comment:
`0` is returned if active era is not determined. Else, we get the `total_issuance` and work out `staked` as a `PerThing`. Then plug the values into `compute_inflation`.